model/ftn/w3iogomd.ftn

```diff
@@ -3747,7 +3747,7 @@
 !
 !
 !/OMPG/!$OMP PARALLEL DO
-!PRIVATE(JSEA,ISEA,FACTOR,FACTOR2,KD,FKD,USSCO,M1)
+!PRIVATE(JSEA,ISEA,FACTOR,FACTOR2,KD,FKD,USSCO,M1,MINDIFF,IB)
```

Comment:
NB (line 3789) is assigned to in an OMP and is not private; but also does not appear to be doing anything other than copying value NK...

Reply:
NB has been deleted.

Reply:
FACTOR2 and M1 are not declared in the subroutine, but the other variables could be declared private in the OpenMP loop: PRIVATE(JSEA,ISEA,FACTOR,KD,FKD,USSCO,MINDIFF,IB)

Reply:
Indeed, both FACTOR2 and M1 have been removed from the declaration.